doc/nrf/protocols/matter/index.rst

```diff
@@ -54,7 +56,7 @@ The |NCS| allows you to develop applications with different versions of Matter,
    These changes are listed in the Matter fork section of the |NCS| :ref:`release_notes`.
 
 .. important::
-   For Matter over Thread samples, starting with |NCS| releases after v2.5.2, the default cryptography backend is Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
+   For Matter over Thread samples, starting with |NCS| releases after v2.6.0, the default cryptography backend is Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
    To :ref:`inherit Thread certification <ug_matter_device_certification_reqs_dependent>` from Nordic Semiconductor, you must use the PSA Crypto API backend.
```

Comment:
"backend" is a term we have moved away from as it was used for the `_ALT` based "driver support" that legacy crypto-toobox APIs. Can we fix this to just say "must use PSA Crypto APIs"?

Reply:
Sure, no problem, I will change it as you suggested. :)